### DISCUSSION TITLE
Waschmaschine soll nach Programmende auf "an" bleiben

### DISCUSSION THREAD

**Discussion by sirius-black-2** [camueller/SmartApplianceEnabler#203] [Q&A]
Hallo zusammen,

bei mir passiert es öfter, dass die Waschmaschine nach Programmende stromlos geschalten wird, was zur Folge hat, dass man die Tür nicht öffnen kann. Gesteuert wird die Waschmaschine über den SAE mittels einem Sonoff pow (Tasmota) als HTTP Schalter mit Anlaufstromerkennung (die Mindestlaufzeit steht auf 2h und das Programm scheint auch druchgelaufen zu sein). Leider habe ich im Photovoltaiokforum und hier unter der Rubrik Anlaufstromerkennung nichts gefunden, daher wäre ich für ein paat Tipps dankbar.

**Comment by camueller** [comment]
Die Ursache liegt darin, dass der SAE das Programm-Ende erkennt, obwohl es noch gar nicht beendet wurde. Wenn dann die Maschine wieder mehr Leistung zieht, wird der Anlaufstrom erneut erkannt und das Gerät stromlos geschaltet.

Du musst durch Parametrisierung der Anlaufstromerkennung dafür sorgen, dass das Programmende erst erkannt wird, wenn es tatsächlich soweit ist. Vermutlich musst Du die "Unterschreitungsdauer beim Ausschalten" auf einen höheren Wert setzen.

**Comment by sirius-black-2** [comment]
Vielen Dank, ich hbae jetzt mal die Unterschreitungszeit verdoppelt und werde sehen was passiert!
